<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arinc665Qt::MediaSetManager::MediaSetManagerDialog</class>
 <widget class="QDialog" name="Arinc665Qt::MediaSetManager::MediaSetManagerDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
    <normaloff>:/fa/solid/database.svg</normaloff>:/fa/solid/database.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Media Sets</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableView" name="mediaSets">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="viewMediaSet">
        <property name="toolTip">
         <string>View Media Set</string>
        </property>
        <property name="icon">
         <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
          <normaloff>:/fa/solid/eye.svg</normaloff>:/fa/solid/eye.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importMediaSet">
        <property name="toolTip">
         <string>Import Media Set</string>
        </property>
        <property name="icon">
         <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
          <normaloff>:/fa/solid/arrow-right-to-bracket.svg</normaloff>:/fa/solid/arrow-right-to-bracket.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="importMediaSetXml">
        <property name="toolTip">
         <string>Import Media Set XML</string>
        </property>
        <property name="icon">
         <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
          <normaloff>:/fa/solid/file-import.svg</normaloff>:/fa/solid/file-import.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeMediaSet">
        <property name="toolTip">
         <string>Remove Media Set</string>
        </property>
        <property name="icon">
         <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
          <normaloff>:/fa/solid/trash-can.svg</normaloff>:/fa/solid/trash-can.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="openMediaSetsDirectory">
        <property name="toolTip">
         <string>Open Media Sets Directory</string>
        </property>
        <property name="icon">
         <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
          <normaloff>:/fa/solid/folder-tree.svg</normaloff>:/fa/solid/folder-tree.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settings">
        <property name="toolTip">
         <string>Media Set Manager Settings</string>
        </property>
        <property name="icon">
         <iconset resource="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc">
          <normaloff>:/fa/solid/gear.svg</normaloff>:/fa/solid/gear.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mediaSets</tabstop>
  <tabstop>viewMediaSet</tabstop>
  <tabstop>importMediaSet</tabstop>
  <tabstop>importMediaSetXml</tabstop>
  <tabstop>removeMediaSet</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/arinc665_qt_resources.qrc"/>
  <include location="../../../qt_icon_resources/lib/qt_icon_resources/font_awesome/font_awesome.qrc"/>
 </resources>
 <connections/>
</ui>
